CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 - [FIXED] Renamed attachment property `content_type` from `contentType` to match
   the documentation and CouchDB format.
+- [FIXED] Issue causing crash on startup of iOS apps due to required frameworks not being added to LD_RUNPATH_SEARCH_PATHS.
```

Comment:
Actually I suppose this is necessary for the device case as well, just not sufficient as the other Xcode setup steps are required as well. So I think it's probably fine.